pages/proposals/language/Espanol.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
@mavis I see the "spanish" label translated in the other files, so I guess those would need to be edited to not point to anything Spanish?